perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -606,21 +606,33 @@ sub meta_line {
             $meta = sprintf( _('Reported anonymously at %s'), $date_time );
         }
     } else {
-        if ( $problem->service and $category && $category ne _('Other') ) {
-            $meta = sprintf(
-                _('Reported via %s in the %s category by %s at %s'),
-                $problem->service, $category,
-                $problem->name,    $date_time
-            );
-        } elsif ( $problem->service ) {
-            $meta = sprintf( _('Reported via %s by %s at %s'),
-                $problem->service, $problem->name, $date_time );
-        } elsif ( $category and $category ne _('Other') ) {
-            $meta = sprintf( _('Reported in the %s category by %s at %s'),
-                $category, $problem->name, $date_time );
+      my $problem_name;
```

Comment:
The spacing here has gone a bit wrong – 4 space indents always, please, to make the diff easier :)